bin/release

```diff
@@ -13,6 +13,19 @@ else
   exit 1
 fi
 
+echo "Running the contract test for MI Bridges driving code..."
+RUN_DRIVING_TEST=true bin/rspec spec/features/mi_bridges_driving_spec.rb
```

Comment:
I just had another thought, apologies... should we be running this on the staging environment via `heroku run` so we're testing we have the right buildpacks and gems in the right location?

Reply:
yeah I thought about that. If we go that direction, we will also need to include all of our test-env-specific gems, but I do think it would be good. 🤔

Reply:
Hmmmm, what if we pulled this out of rspec and into a plain old ruby script that lived in `bin` like `bin/test_mi_bridges_driver`so we don't need to pull in all the test gems?

Reply:
@jessieay yeah... This be complicated. I don't know if the additional work to make it run on Heroku is worth the investment?

Reply:
SOme thoughts on how:

1. Emphasize a driver design of failing fast in places of uncertainty?
2. We could build in checks that verify the data we entered is on the page as part of the standard page actions?
3. ???

Reply:
I think 2 is a good idea as we move forward with driving code -- on each step, we should know where we are based on the page title / content

Reply:
I know the proposal is to run it on *staging* but that, for some reason, makes me nervous to make it possible to run anything like this on heroku at all.

Reply:
@jayroh - Aren't we supposed to be running all the driving stuff on Heroku?